AdaSupport/embed
# HOTFIX: custom events

**What**
- Fixes issue where `reset` and `deleteHistory` would break if called right after `start`
- Fixes issue where clicking the embed button before `fetchUnread` response would cause the drawer to open without the iFrame

**How to Test**
- Test calling `deleteHistory` right after `start` (should already be setup for testing in `index.ejs`, just need to uncomment `// adaEmbed.deleteHistory();`)
- Try clicking the bot button as soon as it shows up. Should see Chat load as usual

---
